### PR TITLE
Add tag to the meta descriptions of tag index pages

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Adds the tagname to the meta description of tag index pages.

### CODE REVIEW

File: locales/nl.yml [hunk 1]
```diff
@@ -60,6 +60,8 @@ nl:
       xing: Deel op Xing
     next: "Volgende"
     prev: "Vorige"
+    tags:
+      meta_description: "Lees hier de blogs van Defacto gerelateerd aan "
```
Comment (jessedijkstra) [inline locales/nl.yml]: Maybe: "Blogs van Defacto gerelateerd aan"?

Comment (RonaldDijkstra) [inline locales/nl.yml]: Nice

File: source/blog/tag.html.erb [hunk 2]
```diff
@@ -7,6 +7,7 @@ pageable: true
 ---
 
 <% content_for :page_title, "Tag: #{tagname} - Blog - Defacto" %>
+<% content_for :meta_description, t("blog.tags.meta_description") + tagname %>
```
Comment (sn3p) [inline source/blog/tag.html.erb]: Maybe interpolate the tag name?

```rb
content_for :meta_description, t("blog.tags.meta_description", tagname: tagname)
```

Comment (RonaldDijkstra) [inline source/blog/tag.html.erb]: This doesn't show the tagname in the content_for..

File: locales/nl.yml (hunk 1 above)
Comment (RonaldDijkstra) [inline locales/nl.yml]: Changed it! Thank you

File: source/blog/tag.html.erb (hunk 2 above)
Comment (sn3p) [inline source/blog/tag.html.erb]: You need to include it in the translations:

```yml
meta_description: "Blogs van Defacto gerelateerd aan %{tagname}"
```

Comment (RonaldDijkstra) [inline source/blog/tag.html.erb]: Ahh

**Pull request merged**